### DISCUSSION TITLE
UI any plans to correct it?

### DISCUSSION THREAD

**Discussion by silveroxides** [AlpacaInTheNight/PromptsBrowser#8] [Q&A] [answered]
That is a very invasive UI. Also the fact that it does not adhere to the Gradio theme set makes it even worse.
After I saw no way to fix this without editing it and rewriting the whole UI part I just disabled it.
Are there any plans to correct this so it actually uses the gradio theme set inside SD-Webui?

**Comment by AlpacaInTheNight** [comment, marked as answer]
Yes, I agree that the current implementation fits the system too poorly. It's not so much a desired result as a result of lack of knowledge in python, gradio and time. I can see problems with updates to the application and interaction with other addons. 

I have changed the styles so that they take more values from the gradio theme. Styles and theme support will still need to be worked on though. I've added this to the developer branch for now and I think I'll update the main branch soon after some testing.

In general, I haven't worked much on the visual design of the interface. I just wanted to implement the features I needed and made the interface as a quick fix. The question of the interface is open and any recommendations are welcome. Some of the addon elements will go to the native elements of the application, like setup. Probably will move some functional into a separate application tab.

![theme_support](https://github.com/AlpacaInTheNight/PromptsBrowser/assets/28790695/cb63568b-908d-4b05-9e41-166f04bc36d1)
